zer0-os/zOS
# feat: add icon to create button and improve styling (create conversation flow)

### What does this do?
- fixes width of `create` button
- adds icon to `create` button
- import icon from `zui`
- improves styling as per design

### Why are we making this change?
To fix the width of the button and improve styling as per design.

Before: 

<img width="339" alt="Screenshot 2023-05-26 at 12 02 32" src="https://github.com/zer0-os/zOS/assets/39112648/9dec613c-f936-409e-ac02-b23387661f5c">

After:

https://github.com/zer0-os/zOS/assets/39112648/c1716180-8c46-4979-9d0a-efe683c8f2f3

### How do I test this?
Click to create a new conversation. Click `Start group chat`. Enter a user to start a group chat with. Click `continue`. See `create` button.
